portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,59 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * An array of dog pics' source and alternative text
+ * @const {Array.<{source: string, alt: string}>}
```

Comment:
```suggestion
 * @const {!Array<{source: string, alt: string}>}
```
I don't think you need a "."

Reply:
Understood, thanks!